static/src/javascripts/es6/projects/common/modules/crosswords/clues.js

```diff
@@ -1,9 +1,10 @@
 /* jshint newcap: false */
+import classNames from 'classnames';
+import React from 'react';
+
 import _ from 'common/utils/_';
-import React from 'react/addons';
 
-var classSet = React.addons.classSet,
-    Clue = React.createClass({
+var Clue = React.createClass({
```

Comment:
I think @rich-nguyen decided it was best not to use this, because you have to "manually bind"?